app/views/spree/admin/promotions/index.html.erb

```diff
@@ -64,7 +64,7 @@
             <td><%= promotion.description %></td>
             <td><%= promotion.usage_limit.nil? ? "∞" : promotion.usage_limit %></td>
             <td><%= Spree.t(:current_promotion_usage, count: promotion.credits_count) %></td>
-            <td><%= promotion.expires_at.to_date.to_s(:short_date) if promotion.expires_at %></td>
```

Comment:
This will change the format of the date, which I don't think is intended. 
The issue is caused by Rails 7.1 having removed `to_s` in favor of `to_fs`. Since we'd like to support Rails 6.1+, I think it would be best to use `strftime` instead (just like some other places in the codebase do)